typespec-tests/Generate.ps1

```diff
@@ -55,6 +55,8 @@ $generateScript = {
     $tspOptions += " --option ""@azure-tools/typespec-java.stream-style-serialization=true"""
   } elseif ($tspFile -match "tsp[\\/]union.tsp") {
     $tspOptions += " --option ""@azure-tools/typespec-java.stream-style-serialization=true"""
+  } elseif ($tspFile -match "tsp[\\/]multipart.tsp") {
+    $tspOptions += " --option ""@azure-tools/typespec-java.stream-style-serialization=true"""
```

Comment:
so currently it's user to set stream-style-serialization=true in tspconfig.yaml, instead of us adding the flag automatically if it's multipart?

Reply:
multipart does not depends on stream-style. None of JSON applies in this case.

I did this just to make Alan's life easier (as the test code generated is stream-style, which matches the default on his PR).

Reply:
I see.